wiki/People/Tournament_Committee/en.md

```diff
@@ -26,13 +34,119 @@ Their responsibilities include:
 - Observing the general contest scene and advising the [account support team](/wiki/People/Account_support_team) on a case-by-case basis
 - Regularly reviewing the [Official beatmapping contest support](/wiki/Contests/Official_support) page for revisions when necessary
 
-The panel was first announced to the public on February 28, 2024 through the [Mapping Discussion](https://osu.ppy.sh/community/forums/topics/1890429?n=1) forum.
+The panel was first announced to the public on February 28, 2024 through the [Mapping Discussion](/community/forums/topics/1890429) forum.
+
+## Tournament Committee Workflow
+
+### Tournament Reports
+
+Tournament reports are the preferred way for players and tournament staff to report infringements of the [official tournament support](wiki/Tournaments/Official_support) expectations and eligibility. When a new report is sent in, all Tournament Committee members are automatically notified. Members will read the report and determine whether to pursue action based on the report's contents. Such action includes but is not limited to:
+
+- Marking the tournament for investigation pending further reports if this report alone is not sufficient to warrant further action.
+- Further investigation and context gathering.
+- Reaching out to involved parties for further information about the events in the report.
+- Voting on whether to withhold a tournament badge.
+- Voting on whether to administer a tournament ban.
+- Voting on whether to administer a tournament staffing ban.
+
+### Tournament Review
+
+Once a tournament has concluded and has requested a badge prize, two random members from the Tournament Committee are assigned to review the tournament. The assigned members will ensure the following:
+
+- The format adheres to [official support](wiki/Tournaments/Official_support) guidelines or received an official approval.
+- The tournament theme does not violate community rules.
+- Any regional registration restrictions are geographically and/or culturally consistent.
+- Any unofficial screening process is open-source with a publicly available dataset.
```

Comment:
I feel the need to comment that in [this draft](https://github.com/ppy/osu-wiki/commit/1b6d2e0490cde8decff7c070f074395f9a0c9218#diff-35eeda5851b226096d81386f70bacafc5f63e4b140adb48060969d2c9987c7beR107) of changes regarding 'self-screening', we are very intentionally **not** using the word 'screening' anywhere when talking about this process. This is to avoid confusion between screening done by support and *registrant filtering*, which is done by hosts in addition to screening. Having the terms be distinctly separate will help avoid confusion down the road.

This was briefly addressed [here](https://github.com/ppy/osu-wiki/pull/11440#discussion_r1610880272) but I'd like another look into whether this line should make it in this PR at all, or how to word it if so.

Reply:
screening and filtering are synonyms (though screening is more appropriate in this context), so if you want to coin a distinction between them you'd be looking into making a glossary. just using separate words wouldn't be enough

i think "self-screening" is plenty fine here, but when in doubt, be more descriptive

```suggestion
- Any screening process on the tournament host's end is open-source with a publicly available dataset.
```

Reply:
Line is removed until a separate PR for the issue on badge criteria is passed.